StackExchange.Profiling/Storage/DummyStorage.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace StackExchange.Profiling.Storage
+{
+    /// <summary>
+    /// A dummy storage which doesn't actually store the profilers anywhere.
+    /// </summary>
+    public class DummyStorage : IStorage
+    {
+        /// <summary>
+        /// Returns an empty list.
+        /// </summary>
+        public List<Guid> GetUnviewedIds(string user)
+        {
+            return new List<Guid>();
```

Comment:
Returning the same empty list would be a bit better. no reason to initialize a new empty list every time...